pango-view/viewer-render.c

```diff
@@ -712,8 +735,12 @@ parse_options (int argc, char *argv[])
      "Set the margin on the output in pixels",			    "CSS-style numbers in pixels"},
     {"markup",		0, 0, G_OPTION_ARG_NONE,			&opt_markup,
      "Interpret text as Pango markup",					NULL},
-    {"output",		'o', 0, G_OPTION_ARG_STRING,			&opt_output,
-     "Save rendered image to output file",			      "file"},
+    {"output",		0, G_OPTION_FLAG_HIDDEN, G_OPTION_ARG_STRING,    &opt_output_file,
+     "Deprecated",          "file"},
```

Comment:
This is changing how the current utility handles command line options and should be reverted back. Keep `--output` as it was, and make `--output-file` and `--output-format` override it.